crates/zk-por-core/src/account.rs

```diff
@@ -7,3 +13,16 @@ pub struct Account {
     pub equity: Vec<F>,
     pub debt: Vec<F>,
 }
+
+impl Account {
+    /// Gets the account hash for a given account.
+    pub fn get_hash(&self) -> HashOut<F> {
+        let sum_equity = self.equity.iter().fold(F::ZERO, |acc, x| acc + *x);
+
+        let sum_debt = self.debt.iter().fold(F::ZERO, |acc, x| acc + *x);
+
+        let hash = PoseidonHash::hash_no_pad(vec![sum_equity, sum_debt].as_slice());
```

Comment:
the account hash should includes user_id

Reply:
How would we represent the user id in a field?